src/api/app/jobs/staging_project_merge_job.rb

```diff
@@ -0,0 +1,11 @@
+class StagingProjectMergeJob < ApplicationJob
+  queue_as :quick
```

Comment:
You wish this was quick. But in practise this takes several minutes - and I'm not so sure you wouldn't jam the quick queue with that.

Reply:
I don't think there is much we can do about right now. Except to not use delayed jobs. But then api requests might fail with a timeout.

Reply:
Thinking about it... Using `perform_now` might do the trick. I'll check

Reply:
> I don't think there is much we can do about right now. 

We can put it into it's own queue for instance. @eduardoj knows how to start one.

Reply:
I can also do this now, alright?

Reply:
pushed.

Reply:
Sure, thanks:thumbsup: